packages/react-openapi/src/OpenAPISchema.tsx

```diff
@@ -92,6 +95,9 @@ export function OpenAPISchemaProperty(
                             className="openapi-schema-description"
                         />
                     ) : null}
+                    {shouldDisplayExample(schema) ? (
+                        <span className="openapi-schema-example">Example: <code>{schema.example}</code></span>
```

Comment:
I mentioned it a few times already; but until we solve this part we should not merge this PR: this line is wrong.

`schema.example` can be any value; and the way you display it will not work with non-primitive values.

Example taken from the doc https://swagger.io/docs/specification/adding-examples/:

```yaml
paths:
  /users:
    post:
      summary: Adds a new user
      requestBody:
        content:
          application/json:
            schema:      # Request body contents
              type: object
              properties:
                id:
                  type: integer
                name:
                  type: string
              example:   # Sample object
                id: 10
                name: Jessica Smith
      responses:
        '200':
          description: OK
```

With this case, it'll display `Example: [object Object]` which is shit 💩 

This is a very common use case, so we need to handle it properly. I can help over a 30-minute pair programming session, or you can iterate on it.

Reply:
To explain more and why we can't merge it as is: you are solving it for one customer by handling only primitives; every customers with non-primitive examples (like in the one above) will be `Example: [object Object]` and will consider worst than not having it. Basically, this PR will be a net-negative for all users.

Reply:
Haa I see that you added the non-prmitive... ok let's merge but fix the boolean case first:

Otherwise it'll display `Example: `